Code/Engine/Foundation/Containers/Implementation/HashTable_inl.h

```diff
@@ -325,7 +325,7 @@ bool ezHashTableBase<K, V, H>::Insert(CompatibleKeyType&& key, CompatibleValueTy
 {
   Reserve(m_uiCount + 1);
 
-  ezUInt32 uiIndex = H::Hash(key) % m_uiCapacity;
+  ezUInt32 uiIndex = H::Hash(key) & (m_uiCapacity - 1);
```

Comment:
How will this work, if capacity is not a power of two? @C-Core

Reply:
We ensure that we always have power of two sizes in reserve. @Ingrater the compact method doesn't ensure it atm, can you do the same logic there as in reserve.